### ISSUE TITLE
TimeZoneLoader::loadVTimeZone does not close InputStream

### ISSUE THREAD

**Issue by symVmueller** [ical4j/ical4j#249]
**Description**
TimeZoneLoader::loadVTimeZone retrieves an InputStream, that never is closed. This leads to Warnings when undeploying an application that uses ical4j in glassfish.

**Expected behavior**
No Warnings. The InputStream is closed after using it.

**Environment:**
 - OS: Ubuntu 16.04
 - Java version: 8_u161
 - Application server: Payara 4.174
 - ical4j version: 3.0.0

**Additional context**
Partial Stack Trace:
```
[javax.enterprise.system.util] [tid: _ThreadID=98 _ThreadName=AutoDeployer] [timeMillis: 1529675310657] [levelValue: 900] [[
  Input stream has been finalized or forced closed without being explicitly closed; stream instantiation reported in following stack trace
java.lang.Throwable
        at com.sun.enterprise.loader.ASURLClassLoader$SentinelInputStream.<init>(ASURLClassLoader.java:1284)
        at com.sun.enterprise.loader.ASURLClassLoader$InternalJarURLConnection.getInputStream(ASURLClassLoader.java:1392)
        at java.net.URL.openStream(URL.java:1045)
        at net.fortuna.ical4j.model.TimeZoneLoader.loadVTimeZone(TimeZoneLoader.java:99)
        at net.fortuna.ical4j.model.TimeZoneRegistryImpl.getTimeZone(TimeZoneRegistryImpl.java:181)
...
```

**Comment by benfortuna** [comment]
Fix applied in PR #251

**Comment by benfortuna** [comment]
Fix applied, closing issue.
